notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1567 @@
+{
```

Comment:
<p>"The Taxi dataset contains trips among every boroughs in NYC, while the Citi Bike dataset has trips in between Manhattan and some part of Brooklyn and Bronx. "</p><p>-&gt; </p><p>The Taxi dataset contains trips for all NYC boroughs, while the Citi Bike dataset has trips between Manhattan and parts of Brooklyn and the Bronx. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>